common/src/main/java/com/github/teamfusion/summonerscrolls/common/entity/goal/OwnerHurtByTargetGoal.java

```diff
@@ -0,0 +1,50 @@
+package com.github.teamfusion.summonerscrolls.common.entity.goal;
+
+import com.github.teamfusion.summonerscrolls.common.entity.ISummon;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.Mob;
+import net.minecraft.world.entity.ai.goal.target.TargetGoal;
+import net.minecraft.world.entity.ai.targeting.TargetingConditions;
+
+import java.util.EnumSet;
+
+public class OwnerHurtByTargetGoal extends TargetGoal {
```

Comment:
Doesn't this also already exist as a Vanilla goal? I could be wrong on both of these, but I could've sworn they do, since wolves would use them.

Reply:
Same as above